src/main/java/org/gridsuite/explore/server/services/ExploreService.java

```diff
@@ -60,9 +62,16 @@ public ExploreService(
         this.parametersService = parametersService;
     }
 
-    public void createStudy(String studyName, UUID caseUuid, String description, String userId, UUID parentDirectoryUuid, Map<String, Object> importParams, Boolean duplicateCase) {
+    @Data
+    @AllArgsConstructor
+    public static class CaseInfo {
```

Comment:
Should use record for DTO instead of class

Reply:
fixed